src/main/java/uk/gov/justice/hmpps/casenotes/config/AuthAwareTokenConverter.kt

```diff
@@ -9,8 +9,11 @@ import org.springframework.security.oauth2.server.resource.authentication.JwtAut
 import uk.gov.justice.hmpps.casenotes.config.SecurityUserContext.UserIdUser
 
 class AuthAwareTokenConverter : Converter<Jwt, AbstractAuthenticationToken> {
-  override fun convert(jwt: Jwt): AbstractAuthenticationToken =
-      AuthAwareAuthenticationToken(jwt, jwt.claims["user_id"] as String, extractAuthorities(jwt))
+  override fun convert(jwt: Jwt): AbstractAuthenticationToken {
+    val optionalUserId = jwt.claims["user_id"]
```

Comment:
so `user_id` can be null if client credentials, so use `subject` instead which is guaranteed to be populated